x-pack/plugins/data/kibana.json

```diff
@@ -0,0 +1,13 @@
+{
+  "id": "data_xpack",
```

Comment:
I think this should be data_enhanced as per @lukeelmers and teams decidion

Reply:
np, renamed to `data_enhanced`

Reply:
yep it sounds like `data_enhanced` has been the consensus so far, so this works for me!